.gitignore

```
@@ -7,3 +7,4 @@ target/
 *.tmp
 java_pid*
 dump/test-basedir
+GSOC external ressources/
```

This probably also mandates a directory name change
```suggestion
GSOC external resources/
```